hs_composite_resource/models.py

```diff
@@ -132,43 +127,17 @@ class name of that matching aggregation type.
             files_in_sub_folders = ResourceFile.list_folder(self, folder=irods_path,
                                                             sub_folders=True)
             if files_in_sub_folders:
-                return FileSetLogicalFile.__name__
+                return True
 
-            return None
+            return False
         if store[0]:
-            # there are folders under dir_path as well as files - only FileSet can bet set
-            return FileSetLogicalFile.__name__
-
-        if len(files_in_folder) > 1:
-            # check for geo feature
-            aggregation_type_to_set = GeoFeatureLogicalFile.check_files_for_aggregation_type(
-                files_in_folder)
-            if aggregation_type_to_set:
-                return aggregation_type_to_set
-
-            # check for raster
-            aggregation_type_to_set = GeoRasterLogicalFile.check_files_for_aggregation_type(
-                files_in_folder)
-            if aggregation_type_to_set:
-                return aggregation_type_to_set
-            return FileSetLogicalFile.__name__
+            # there are folders under dir_path as well as files - FileSet can bet set
+            return True
+
+        if len(files_in_folder) > 0:
+            return True
         else:
-            # check for raster
-            aggregation_type_to_set = GeoRasterLogicalFile.check_files_for_aggregation_type(
-                files_in_folder)
-            if aggregation_type_to_set:
-                return aggregation_type_to_set
-            # check for NetCDF aggregation type
-            aggregation_type_to_set = NetCDFLogicalFile.check_files_for_aggregation_type(
-                files_in_folder)
-            if aggregation_type_to_set:
-                return aggregation_type_to_set
-            # check for TimeSeries aggregation type
-            aggregation_type_to_set = TimeSeriesLogicalFile.check_files_for_aggregation_type(
-                files_in_folder)
-            if aggregation_type_to_set:
-                return aggregation_type_to_set
-            return FileSetLogicalFile.__name__
+            return False
```

Comment:
I believe the logic in lines 124 to 140 can be combined/simplified/corrected. For example, ```if not files_in_folder``` is equal to the effect on lines 139-140, so at least lines 139-140 are not needed.